src/components/HostMetaDataCard.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { ApolloError } from "apollo-client";
+import { Skeleton } from "antd";
+import { ErrorWrapper } from "components/ErrorWrapper";
+import styled from "@emotion/styled/macro";
+import Card from "@leafygreen-ui/card";
+
+interface Props {
+  title?: string;
+  error: ApolloError;
+  loading?: boolean;
+}
+
+export const HostMetaDataCard: React.FC<Props> = ({
+  children,
+  error,
+  loading,
+}) => (
+  <SiderCard>
+    {loading && !error && (
+      <Skeleton active title={false} paragraph={{ rows: 0 }} />
```

Comment:
0 rows will make hide the skeleton. we should add some rows so we can show it while it's loading